### DISCUSSION TITLE
Users and Maintainers

### DISCUSSION THREAD

**Discussion by hallyn** [lxc/lxcri#64] [General]
We've been assuming that this was maintained and - more importantly - in use by @r10r .  However, it appears that it is not.

Could you confirm, @r10r , whether you are using this in production anywhere?

Could anyone else who's using this in production speak up?

**Comment by r10r** [comment]
@hallyn No we're not using it in production.

**Reply by hallyn** [reply]
Thanks.

**Comment by hallyn** [comment]
If noone is using and noone is maintaining this project, shall we archive it?

Let's consider that in a week, if noone replies showing interest.

**Comment by zwindler** [comment]
I'm not using it in production but I would have liked to try it at least

I have tried to build it (failed) and opened issues
